config/config_custom.sh

```diff
@@ -16,3 +16,4 @@ export   MAKE_CLEAN=N
 export DOWNLOAD_ONLY=N
 export STACK_EXIT_ON_FAIL=Y
 export WGET="wget -nv"
+
```

Comment:
un-necessary

Reply:
Sorry, the extra line was unintentional.